elasticdl/python/common/ndarray.py

```diff
@@ -50,7 +50,7 @@ def tensor_to_ndarray(tensor_pb):
     return arr
 
 
-def ndarray_to_tensor(arr, indices=None):
+def ndarray_to_tensor(arr, name=None, indices=None):
```

Comment:
We need to add a Tensor Python data structure here. This could be done in the next PR.

Reply:
Yes. I will add this data structure immediately after this PR is merged.